eng/common/docgeneration/Generate-DocIndex.ps1

```diff
@@ -119,7 +119,9 @@ function GenerateDocfxTocContent([Hashtable]$tocContent, [String]$lang, [String]
         $serviceName = $serviceMapping.Value[0]
         $displayName = $serviceMapping.Value[1]
 
-        $fileName = ($serviceName -replace '\s', '').ToLower().Trim()
+        # handle spaces in service name, EG "Confidential Ledger"
+        # handle / in service name, EG "Database for MySQL/PostgreSQL". Leaving a "/" present will generate a bad link location.
+        $fileName = ($serviceName -replace '\s', '').Replace("/","").ToLower().Trim()
```

Comment:
@danieljurek it looks like we might want to start having some helper for this as we are doing this in a few places.

Reply:
Can we put check into one regex?
```
$serviceName -replace "\s|\\|\/, ""
```

Reply:
I had already merged this in a few repos before this feedback came in. I'm going to merge this PR.